src/main/java/org/apache/skywalking/banyandb/v1/client/BanyanDBClient.java

```diff
@@ -314,6 +317,31 @@ public void define(Measure measure) throws BanyanDBException {
         MetadataCache.INSTANCE.register(measure);
     }
 
+    /**
+     * Create or update the property
+     *
+     * @param property the property to be stored in the BanyanBD
+     */
+    public void save(Property property) throws BanyanDBException {
+        PropertyStore store = new PropertyStore(checkNotNull(this.channel));
+        try {
+            store.get(property.group(), property.name(), property.id());
+            store.update(property);
+        } catch (BanyanDBException ex) {
+            // TODO: polish "404" response code in BanyanDB
```

Comment:
Do you tend to have the server return 404 on missing properties?

Reply:
Yes. I think so. Currently, it is `UNKNOWN`. It is not easy to distinguish.

Reply:
https://github.com/apache/skywalking/issues/8829